aiida_zeopp/data/parameters.py

```diff
@@ -123,8 +124,10 @@ def cmdline_params(self, structure_file_name=None, radii_file_name=None):
                 parameter += [v]
 
             # add output file name
-            if k in output_keys:
+            if (k in output_keys) and (k != 'visVoro'):
                 parameter += [self._OUTPUT_FILE_PREFIX.format(k)]
+            elif k.startswith('visVoro'):
```

Comment:
this is not logically correct (it will also trigger when k is not in output_keys).
anyhow, once you've generalized this, you won't need to make an exception for visvoro here anymore

Reply:
Yeah, with the above change it would be back to its original form:
```
if k in output_keys:
                parameter += [self._OUTPUT_FILE_PREFIX.format(k)]
```